lib/lita/handlers/stats.rb

```diff
@@ -0,0 +1,49 @@
+require 'signalfx'
+
+class Stats
```

Comment:
I think if we do `class Stats extend self` or `class Stats < self` you won't have to initialize Stats when using it [here](https://github.com/change/lita-totems/pull/2/files#diff-868c63eb562ba03146634af0ff2cbf9e892e7fd022098d3ae88dd01511befe94R11). 
You would be able to simply call Stats.method where necessary. 
It has been a while since I did Ruby, but that should make this a strictly static class. See the SO below for more info. 
https://stackoverflow.com/questions/5231534/ruby-on-rails-static-method

Reply:
cool, will try it!

Reply:
If all the methods for this class are class methods we can take the shortcut of using `module_function` at the beginning and not having to specify `self.` on all of them.

Reply:
dont we need to change the `class Stats` to `module Stats` to use `module_function`?

Reply:
Sorry, I missed that.But since we're not instantiating a class instance that makes sense to to anyway probably.